packages/core/content-manager/admin/src/pages/App/useModels.js

```diff
@@ -29,12 +30,16 @@ const useModels = () => {
         toggleNotification
       );
 
-      const actionToDispatch = setContentTypeLinks(
-        authorizedCtLinks,
-        authorizedStLinks,
+      const { ctLinks } = runHookWaterfall('cm/mutate-collection-type-links', {
```

Comment:
WYT of creating constatns for the different hook in a dedicated file so that it be a source of documentation for people? Like, they open the "strapi.hooks.js" file and we put key here with information on what the hook does, what we can put in etc?..

Reply:
That's a very good idea!